pkgs/tools/security/yubihsm-connector/default.nix

```
@@ -0,0 +1,28 @@
+{ lib, libusb, buildGoModule, fetchFromGitHub, pkg-config }:
+
+buildGoModule rec {
+  pname = "yubihsm-connector";
+  version = "3.0.2";
+
+  src = fetchFromGitHub {
+    owner = "Yubico";
+    repo = "yubihsm-connector";
+    rev = version;
+    sha256 = "sha256-FQ64tSZN55QpXjMZITzlWOPTKSgnoCpkRngQUQHVavc=";
+  };
+  vendorSha256 = "sha256-kVBzdJk/1LvjdUtLqHAw9ZxDfCo3mBWVMYG/nQXpDrk=";
+
+  patches = [ ./lockfile-fix.patch ];
```

Could you add a comment explaining what it fixes in the lockfile?


```suggestion
  patches = [
    # ???
    ./lockfile-fix.patch
  ];
```

---

Sure.